src/formatting/formatter/datetime.rs

```diff
@@ -100,6 +103,57 @@ impl DatetimeFormatter {
     }
 }
 
+pub(crate) trait TimezoneName {
+    fn timezone_name(datetime: &DateTime<Self>) -> Item
+    where
+        Self: TimeZone;
+}
+
+impl TimezoneName for Tz {
+    fn timezone_name(datetime: &DateTime<Tz>) -> Item {
+        Item::OwnedLiteral(datetime.offset().abbreviation().into())
+    }
+}
+
+impl TimezoneName for Local {
+    fn timezone_name(datetime: &DateTime<Local>) -> Item {
+        let fallback = Item::Fixed(Fixed::TimezoneName);
+        let tz_name = match iana_time_zone::get_timezone() {
+            Ok(tz_name) => tz_name,
+            _ => {
+                debug!("Could not get local timezone");
+                return fallback;
+            }
+        };
+        let tz = match tz_name.parse::<Tz>() {
+            Ok(tz) => tz,
+            Err(e) => {
```

Comment:
It might be a good place to use `let Ok(...) = ... else { ... };`. This is OK too.

Reply:
I can just use that for the tz_name above, since I want the error to log here.

Reply:
Oh, yeah, right :laughing: